test/e2e/playwright.config.ts

```diff
@@ -6,7 +6,7 @@ import { defineConfig, devices } from '@playwright/test';
 export default defineConfig({
     testMatch: /test\/e2e\/specs\/.*(\.spec.ts)/,
 
-    timeout: 45000,
+    timeout: 150000,
```

Comment:
Why did you have to increase the timeout?
(just curious why you needed this change)

Reply:
Ok i see now it is for the deployment preview to be loaded

Reply:
Yes. The page can take some time to be loaded so the timeout was increased.